src/layout/Tabs/config.ts

```diff
@@ -0,0 +1,61 @@
+import { CG, Variant } from 'src/codegen/CG';
+import { CompCategory } from 'src/layout/common';
+
+export const Config = new CG.component({
+  category: CompCategory.Container,
+  rendersWithLabel: false,
+  capabilities: {
+    renderInTable: false,
+    renderInButtonGroup: false,
+    renderInAccordion: false,
+    renderInAccordionGroup: false,
+    renderInTabs: false,
+    renderInCards: false,
+    renderInCardsMedia: false,
+  },
+})
+  .addTextResource(
+    new CG.trb({
+      name: 'ariaLabel',
+      title: 'Aria Label',
+      description: 'The aria label for the tabs',
```

Comment:
Would be nice to describe what an aria label is and how this is being used, instead of just repeating the title in the description with some more words. :pray: I.e. "Label read aloud by screen readers when \<...\>"